src/components/forms/CreateEmailTemplate/server-action.ts

```diff
@@ -0,0 +1,81 @@
+import { TemplateFormState } from '@utils/types';
+import { zodValidationServerAction } from '@utils/zod-validation-server-action';
+import { z } from 'zod';
+import { saveSession } from '@utils/form-actions';
+import { redirect, RedirectType } from 'next/navigation';
+
+const $CreateEmailTemplateSchema = z.object({
+  emailTemplateName: z
+    .string({ message: 'Enter a template name' })
+    .min(1, { message: 'Enter a template name' }),
+  emailTemplateSubjectLine: z
+    .string({ message: 'Enter a template subject line' })
+    .min(1, { message: 'Enter a template subject line' }),
+  emailTemplateMessage: z
+    .string({ message: 'Enter a template message' })
+    .min(1, { message: 'Enter a template message' }),
```

Comment:
in Notify core this has an upper limit of 100,000

Reply:
Removed since the prototype didn't show a character limit thingy.  Can add a limit to match core. Will check with UCD if we want a character limit being displayed 👍